### ISSUE TITLE
Docs seems outdated

### ISSUE THREAD

**Issue by sielay** [krakenjs/adaro#103]
Hi

Docs say you can do

``` javascript
var adaro = require('adaro'),
instance = adaro(options);
```

and those options can pass helpers. But it's not a case anymore, as export from adaro module is not a function. Either this

``` javascript
adaro.dust({ ... })
```

won't pass helpers, only way that seems working for me is

``` javascript
var dust = adaro.dust({});
dust.dust.helpers = ...
```

**Comment by grawk** [comment]
Hi @sielay. Can you please point out the exact document and section that is misleading? This module's README doesn't appear to show such an example.

**Comment by sielay** [comment]
In your readme - you take module exports of `adaro` to `adaro` var (example 1)

```
var adaro = require('adaro');
```

You pass helpers via options (example 2)

```
var options = {
  helpers: [
    //NOTE: function has to take dust as an argument.
    //The following function defines @myHelper helper
    function (dust) { dust.helpers.myHelper = function (a, b, c, d) {} },
    '../my-custom-helpers',   //Relative path to your custom helpers works
    'dustjs-helpers',   //So do installed modules
    {
      name: '../my-custom-helpers/helper-to-render-data-with-args',
      // or use this signature if you need to pass in additional args
      arguments: { "debug": true }
    }
  ]
};
```

And now this is version of current readme

```
app.engine('dust', adaro.dust(options));
```

and that is version of doc in package on npm

```
app.engine('dust', adaro(options)); 
```

Change happened here https://github.com/krakenjs/adaro/commit/fd7d42f0e98182ef545a9b4426f68ac25401714b but seems to not be reflected in npm.

Version in `npm` will throw error that adaro is not a function (example 1)

Still, if I pass helpers I don't have them registered then in dust instance in adaro object given by invoking `adaro(options)`. To make them appear I had to do something like

```
let dust = adaro.dust({
        cache: false
    });

    dust.dust.helpers = helpers;

    server.engine('dust', dust);
```

This finally worked for me.

I think both issues may be due to npm not updated.

**Comment by grawk** [comment]
The first issue is documentation related. I didn't look at what was showing on npm until now, but I see that since v1.0.1 was published there are a couple changes to the README. I'll publish those as v1.0.2.

Regarding your issue registering helpers, that looks like a separate one to me.

Let's address the documentation with this issue and deal with the helpers issue separately.

**Comment by grawk** [comment]
Resolved with the publish of `v1.0.2`.
